applications/serial_lte_modem/src/twi/slm_at_twi.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef SLM_AT_TWI_
+#define SLM_AT_TWI_
+
+/**@file slm_at_twi.h
+ *
+ * @brief Vendor-specific AT command for TWI service.
+ * @{
+ */
+
+#include <zephyr/types.h>
+#include "slm_at_host.h"
+
```

Comment:
I think that these two headers are not needed here

Reply:
You are correct. Removed.